src/app/components/bitcoin-transaction-item/bitcoin-transaction-item.tsx

```diff
@@ -77,17 +77,18 @@ export function BitcoinTransactionItem({ transaction }: BitcoinTransactionItemPr
 
   const title = inscriptionData ? `Ordinal inscription #${inscriptionData.number}` : 'Bitcoin';
   const increaseFeeButton = (
-    <IncreaseFeeButton
+    <TransactionActionButton
+      label="Increase fee"
       isEnabled={isEnabled}
       isSelected={pathname === RouteUrls.IncreaseBtcFee}
-      onIncreaseFee={onIncreaseFee}
+      onClick={onIncreaseFee}
     />
   );
 
   return (
     <TransactionItemLayout
       openTxLink={openTxLink}
-      rightElement={isEnabled ? increaseFeeButton : undefined}
+      actionButtonGroupElement={isEnabled ? increaseFeeButton : undefined}
```

Comment:
Not sure why `rightElement` needs to be renamed bc here a `group` isn't being passed? 🤔

Reply:
> Not sure why `rightElement` needs to be renamed bc here a `group` isn't being passed? 🤔

It is mainly because even though this PR handles only STX cancelling the original issue addresses both STX and BTC tx cancelling issues. So once a solution is up for BTC Tx cancelling there would be a group and the `TransactionItemLayout` is used both in STX and BTC components

Reply:
Thanks Aman. I think it's better to do only what you need to do now, and then you could refactor this if / when adding the next part of cancelling the BTC transaction. 

Doing it now just adds more code to this work that is un-related to the actual task

Reply:
> Thanks Aman. I think it's better to do only what you need to do now, and then you could refactor this if / when adding the next part of cancelling the BTC transaction.
> 
> Doing it now just adds more code to this work that is un-related to the actual task

Alright! will do that

Reply:
Can you link the designs you are following here for the changes? Has a designer approved them moving? They look a bit odd imo. cc @mica000

Reply:
> Can you link the designs you are following here for the changes? Has a designer approved them moving? They look a bit odd imo. cc @mica000

This was the only reference i had in the issue
https://user-images.githubusercontent.com/33360391/224333152-ce1d1f78-53dc-4eb4-9148-b3504a6fd4c8.png

Trying this i would have to move the tx status specifically when its `pending` also this looks more congested in my opinion.

Reply:
There always needs to be a design approval.

Reply:
Hi @Aman-zishan, Michelly over here, designer at Leather :) 

Thank you for your input in this issue! Much appreciated!

From the images that you share above, which one is the most up to date?
If it's this one, then it's good to go!
![image](https://github.com/leather-wallet/extension/assets/119523541/5039208f-7ec3-4649-ad07-3781e2d9b46e)

The second image would be an issue because it would break our layout since 3 lines doesnt align well in our case the same way it does for Metamask.

Reply:
@fbwoolf 

```js
    <Pressable onClick={openTxLink} my="space.02">
      <ItemLayout
        flagImg={txIcon && txIcon}
        titleLeft={txTitle}
        captionLeft={
          <HStack alignItems="center">
            <Caption
              overflow="hidden"
              textOverflow="ellipsis"
              maxWidth={{ base: '160px', md: 'unset' }}
            >
              {txCaption}
            </Caption>
            {rightElement ? rightElement : txStatus}
          </HStack>
        }
        titleRight={!rightElement && <styled.span textStyle="label.02">{txValue}</styled.span>}
      />
    </Pressable>
    
Refactoring the `transactionItemLayout` like this gets the suggested layout. The problem is this doesn't look right (even before refactoring) as the `rightElement` is passed to `titleRight`, which is anti pattern. I'm not sure if refactoring this would add to the scope of this PR, but should I change this layout as a whole to include the buttons, or should I leave it like this?

Reply:
Tbh, I don't know what you mean here w/out images of `doesn't look right` ...and I'm unaware of what you mean as an `anti-pattern`? Would it be best for someone on the team to take this over now as there appears to be more design considerations to consider than were originally communicated? We are moving UI components to another repo so we shouldn't be making changes, if needed, to the `ItemLayout` component in the extension directly. I am unaware rn if that is needed.

Reply:
> Tbh, I don't know what you mean here w/out images of `doesn't look right` ...and I'm unaware of what you mean as an `anti-pattern`? Would it be best for someone on the team to take this over now as there appears to be more design considerations to consider than were originally communicated? We are moving UI components to another repo so we shouldn't be making changes, if needed, to the `ItemLayout` component in the extension directly. I am unaware rn if that is needed.

By anti-pattern i mean not visually but rather in code style; that a button element was passed to a `titleRight` attribute instead of a text element which it should be based on the attribute naming. And i agree that since leather UI is being integrated this component can be rewritten in this situation to fit in the new layout for the buttons now. Both works for me, i have already marked the PR as draft, but if its easier for the team to take over i am okay with that too.

Reply:
The naming is just to establish the location in the layout, each of `titleRight` etc can just receive a `ReactNode` to render. I don't know of anything specifying it has to be `text`? We can just change the props to `upperRight` and `lowerRight` etc if helps clarify, but I don't think that needs to be done with this work.

Reply:
In testing this PR @mica000 why are we relocating these buttons? They were in the upper right corner, is this intentional? I know the alignment here is off, but do we want them in the lower left?

![Screenshot 2024-06-20 at 1 48 15 PM](https://github.com/leather-io/extension/assets/6493321/e2e0b8fb-a4e9-4ea5-8d74-914366fa3ad6)

Reply:
I think maybe there was a miscommunication here? I meant the upper right corner of the asset row where the balance is typ seen, that is where the current increase fee button is located:

<img width="915" alt="Screenshot 2024-06-21 at 11 04 35 AM" src="https://github.com/leather-io/extension/assets/6493321/d06dc07c-0599-478d-8442-d5ab2aaa1611">

Reply:
> I think maybe there was a miscommunication here? I meant the upper right corner of the asset row where the balance is typ seen, that is where the current increase fee button is located:
> 
> <img alt="Screenshot 2024-06-21 at 11 04 35 AM" width="915" src="https://private-user-images.githubusercontent.com/6493321/341822891-d06dc07c-0599-478d-8442-d5ab2aaa1611.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTkxMzg3MzgsIm5iZiI6MTcxOTEzODQzOCwicGF0aCI6Ii82NDkzMzIxLzM0MTgyMjg5MS1kMDZkYzA3Yy0wNTk5LTQ3OGQtODQ0Mi1kNWFiMmFhYTE2MTEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDYyMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA2MjNUMTAyNzE4WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NjEyMWUyYzNkYmI0YTQ3MTE3YWRkMTZiOTFmMTFkNDM2ZTNmNDc3ODczMjM1MDliOTNjNTUzMmFmOTYyYzg4YSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.n9yOmjM9IGLxIlhgFi60vR_tXIKsQXrfc_i8-eY6yVs">

Yes, it is, and that is what i was trying t oconvey that the layout is off in that aspect, and @mica000 pointed out here https://github.com/leather-io/extension/pull/5536#discussion_r1645068054 that i can use the new layout, but the thing is this would require renaming of `titleRight` to something like `rightElement` and even if i don't rename it does require a restructure of `Itemlayout` component to include the cancel button with increase fee button. `Itemlayout` is used throughout in the app. This is the part where i'm confused 🤔. I would like to finish up this work and move on to the BTC one , but the work on UI library thing recently also is frequent.

Reply:
I found a way to fix this issues with least changes to the layout without affecting the current code

<img width="379" alt="Screenshot 2024-06-24 at 5 44 41 PM" src="https://github.com/leather-io/extension/assets/55238388/00e3ffb2-09e6-46ff-b378-0a22591ea95d">
<img width="379" alt="Screenshot 2024-06-24 at 5 44 53 PM" src="https://github.com/leather-io/extension/assets/55238388/252ed692-dcbf-4601-9dd5-ff5f95f5ad90">
<img width="379" alt="Screenshot 2024-06-24 at 5 45 19 PM" src="https://github.com/leather-io/extension/assets/55238388/9cdd2c47-f0ad-4e0e-8354-26f00c7163f6">

Renaming label from 'Increase Fee' to 'Speed Up' increased space availability. Disabled showing the txStatus when the transaction is pending. Let me know if this works. @mica000 cc: @fbwoolf

Reply:
@Aman-zishan you added this here https://github.com/leather-io/extension/issues/5570 that is good, seems indeed outside the scope of this PR. I added a comment there.

Reply:
@fbwoolf For the use cases that @314159265359879 is referring I believe the right way to solve would be to truncate the content on the left – Wondering now,  would this be feasible in this ticket or we are looking for an easier way of solving this until approval UX is done?

Also another possible alternatives:

**Icon + Tooltip**
> Could we get away with just showing the logos without text? @mica000
I believe we could but in this case we would need to add a tooltip on both buttons saying "cancel" & "increase fee" – Question for @Aman-zishan would that increase too much the scope?

**Smaller wording**
1#) raise fee (Preference), 2#) boost fee, 3#) up fee

Reply:
imo adding a tooltip or small wording can be considered within this scope as i just have to modify the newly added component rather than changing the layout code. Personally i like the small wording solution.

Reply:
Here is the proposal for review @markmhendrickson @fbwoolf
Im using 390px as this is the size of the extension.

<img width="1126" alt="image" src="https://github.com/leather-io/extension/assets/119523541/c9f3d18a-b661-46de-a2ba-cd26bea9d9d6">

Reply:
Cool so it seems like the proposal is approved. 

@Aman-zishan I replaced the old buttons for the ghost button we are currently using in the design system, which already has all the states in it.

Reply:
> Cool so it seems like the proposal is approved.
> 
> @Aman-zishan I replaced the old buttons for the ghost button we are currently using in the design system, which already has all the states in it.

Alright! ill look into that, also i am assuming that a tool tip component also exists within the leather UI library?

Reply:
Yep, it does.

Reply:
> Cool so it seems like the proposal is approved.
> 
> @Aman-zishan I replaced the old buttons for the ghost button we are currently using in the design system, which already has all the states in it.

@mica000 
<img width="389" alt="Screenshot 2024-06-28 at 11 46 14 AM" src="https://github.com/leather-io/extension/assets/55238388/dca4b241-8628-458b-be17-981ac3f82425">

Is the icon replacement deployed to the UI library? I still get the old icons from the UI library, with the cancel icon being slightly larger by default.

cc: @fbwoolf

Reply:
We didn't replace the icons in the UI library so that's something that would need to be done as part of this also. Adding / replacing icons in `@leather-io/ui`

Reply:
> We didn't replace the icons in the UI library so that's something that would need to be done as part of this also. Adding / replacing icons in `@leather-io/ui`

Is it something that is in the list for the ongoing UI library updates? So that i can rebase and focus on the integration tests now?

Reply:
I don't have it on my radar right now. We can open a task for it so you could tackle that after solving the integration tests? Then it would be a case of just swapping out the component when ready.

Reply:
> I don't have it on my radar right now. We can open a task for it so you could tackle that after solving the integration tests? Then it would be a case of just swapping out the component when ready.

Alright! that works

Reply:
I double checked and we already have these icons. Did you try passing `variant="small"` to reduce the size?

Reply:
> I double checked and we already have these icons. Did you try passing `variant="small"` to reduce the size?

for the close icon? yes i did it still was different

Reply:
@pete-watters Could you let me know what is the icon name exported from the ui library for the icons used by @mica000, i couldn't find the icon used in the above mockup and the close icon currently used is still bigger even with variant set to small.